src/mutation/__tests__/RelayMutationQuery-test.js

```diff
@@ -305,6 +305,96 @@ describe('RelayMutationQuery', () => {
       };
     });
 
+    it('refetches the whole range when the matching rangeBehavior is set to REFETCH', () => {
```

Comment:
Our internal linter is probably going to complain about this long line. Can you rephrase it as something like:

    it('refetches the whole range when the rangeBehavior is REFETCH', () => {